griddedio/FieldBundleRead.F90

```diff
@@ -58,7 +58,8 @@ subroutine MAPL_create_bundle_from_metdata_id(bundle,metadata_id,file_name,only_
          character(len=:), allocatable :: units,long_name
 
          collection => DataCollections%at(metadata_id)
-         metadata => collection%find(trim(file_name))
+         metadata => collection%find(trim(file_name), rc=status)
```

Comment:
Please remember to start using `__RC__` for these cases.   It helps to unclutter the code.  Not worth a separate commit in just this instance, but all future changes in MAPL should try to follow this best practice.

Reply:
I have changed them to __RC__ .  Now pFIO depends on esmf, which should be fine